test/sql/insert_sql_test.cpp

```diff
@@ -619,6 +627,86 @@ TEST_F(InsertSQLTests, NonExistentTable) {
   txn = txn_manager.BeginTransaction();
```

Comment:
You still have a memory leak. This transaction is not terminated.

Reply:
Fixed.